platform-coop-toolkit/coop-map-directory-index
# feat: add organization language (resolve #72)

## Description

Adds Operating language field to Organizations.

## Steps to test

1. Check out branch.
2. Run migrations: `./manage.py migrate`
3. Log in to admin.

**Expected behavior:** Organizations now include an "Operating language" field.

## Additional information

Not applicable.

## Related issues

- Requires https://github.com/platform-coop-toolkit/coop-map-directory-index/pull/66.
- Resolves #72.
